gnovm/pkg/integration/coverage.go

```diff
@@ -0,0 +1,66 @@
+package integration
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+	"testing"
+
+	"github.com/rogpeppe/go-internal/testscript"
```

Comment:
Can you refactor the code so that this import is only used in unit tests (`_test.go`)?

Your helpers are good, but instead of configuring `p *testingscript.Params` from a non-test file, you can make this helper return callbacks. These callbacks can then be used to configure testscript from a test file.

Reply:
Actually, it's not that bad because the `integration` package should only be used for tests and tools, not in production. It's not very important.